test/unit/proxy_test.rb

```diff
@@ -73,6 +74,36 @@ def test_policy_chain
       assert_equal policy_chain, proxy.policy_chain
     end
 
+
+    def test_policy_chain_with_backend_apis
+      rolling_updates_on
+
+      account = FactoryBot.create(:simple_provider)
+      service = FactoryBot.create(:simple_service, account: account, act_as_product: true)
+      null_backend_api = FactoryBot.create(:backend_api, account: account, private_endpoint: 'https://foo.baz')
+      null_backend_api.update_columns(private_endpoint: '')
+      backend_api1 = FactoryBot.create(:backend_api, account: account, private_endpoint: 'https://private-1.example.com')
+      backend_api2 = FactoryBot.create(:backend_api, account: account, private_endpoint: 'https://private-2.example.com')
+      FactoryBot.create(:backend_api_config, path: '/null', backend_api: null_backend_api, service: service)
+      FactoryBot.create(:backend_api_config, path: '/foo', backend_api: backend_api1, service: service)
+      FactoryBot.create(:backend_api_config, path: '/bar', backend_api: backend_api2, service: service)
+      policy_chain =  [
+        {"name"=>"routing", "version"=>"builtin", "enabled"=>true,
+          "configuration"=>{
+            "operations"=>[
+              {"url"=>"https://echo-api.3scale.net:443", "condition"=>{"operations"=>[{"match"=>"path", "op"=>"matches", "value"=>"/.*"}]}},
```

Comment:
Just to be sure, this will always match, so the request will never be proxied to the 2 API backends below. Is that what you need in this example?

The routing policy will route to the first URL that matches.

Reply:
Correct, This is because of the first backend api that has "/" in the routing. We allow that but it would be better to have ordering of `path` so that we can use sub-routing like {`/foo` and `/foo/bar`} or simply  {`/` and `/foo`}

But that would necessarily implies the need ordering of path. I do not know if we have to fix that now (at least not in this PR)